libraries/3d-tools/README.md

```diff
@@ -0,0 +1,17 @@
+# Esri 3d Widget Components
+
+Esri's JavaScript 4.x API contains a series of ready-to-use UI widgets. Included in this library are some of Esri's 3d widgets wrapped in a VertiGIS Studio Web Web component so that they can be used in a VertiGIS Studio Web application.
```

Comment:
Just confirming that `Studio Web Web Component` is intentional

Reply:
Nope.